docusaurus.config.js

```diff
@@ -120,6 +118,9 @@ module.exports = {
       },
       copyright: `Copyright © ${new Date().getFullYear()} The Neon Contributors`,
     },
+    prism: {
```

Comment:
This is how [Docusaurus allows adding non-default language support for code blocks](https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages).

We originally had vendored a forked copy of the CodeBlock theme to add Rust and TOML support, but that's not necessary.